### PR TITLE
2.2_Branch: fix POC command bug [skip ci]

### PR DESCRIPTION
**Pull request by chesterxgchen**
Issue is caused by unpack_archive only to current directory not dest poc directory

### CODE REVIEW

File: nvflare/lighter/poc.py [hunk 1]
```diff
@@ -32,11 +32,17 @@ def clone_client(num_clients: int, dest_poc_folder):
     shutil.rmtree(src_folder)
 
 
-def unpack_poc() -> bool:
+def unpack_poc(dest_poc_folder) -> bool:
     file_dir_path = pathlib.Path(__file__).parent.absolute()
     poc_zip_path = file_dir_path.parent / "poc.zip"
     try:
-        shutil.unpack_archive(poc_zip_path)
+        import tempfile
```
Comment (IsaacYangSLA) [inline nvflare/lighter/poc.py:39]: Suggest:

try:
    with tempfile.TemporaryDirectory() as tmp_dir:
        ....
except shutil.ReadError:


Current codes will skip temp_dir.cleanup() when shutil.ReadError happens.

File: nvflare/lighter/poc.py [hunk 2]
```diff
@@ -32,11 +32,16 @@ def clone_client(num_clients: int, dest_poc_folder):
...
+
+        with tempfile.TemporaryDirectory() as tmp_dir:
+            shutil.unpack_archive(poc_zip_path, extract_dir=tmp_dir.name)
```
Comment (IsaacYangSLA) [inline nvflare/lighter/poc.py:42]: tmp_dir is the name of the temp directory, so it's a string.  I think you don't need tmp_dir.name.

Comment (IsaacYangSLA) [reply nvflare/lighter/poc.py:42]:
```
$ python3
Python 3.10.4 (main, Jun 29 2022, 12:14:53) [GCC 11.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import tempfile
>>> with tempfile.TemporaryDirectory() as tmp_dir:
...   print(tmp_dir)
...   print(tmp_dir.name)
... 
/tmp/tmpyk0leofm
Traceback (most recent call last):
  File "<stdin>", line 3, in <module>
AttributeError: 'str' object has no attribute 'name'
```

**Review by IsaacYangSLA — APPROVED** [review]
Thanks.  LGTM.

**Review by YuanTingHsieh — APPROVED** [review]

**Pull request merged**